src/bin/nydus-image/deduplicate.rs

```diff
@@ -0,0 +1,361 @@
+// 2022 Nydus Developers.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+//! Deduplicate for Chunk
+
+use anyhow::{Context, Result};
+use nydus_api::ConfigV2;
+use nydus_builder::Tree;
+use nydus_rafs::metadata::RafsSuper;
+use nydus_storage::device::BlobInfo;
+use rusqlite::{params, Connection};
+use std::error::Error;
+use std::fs;
+use std::path::Path;
+use std::sync::{Arc, Mutex};
+pub trait Database {
+    fn create_chunk(&self) -> Result<(), Box<dyn Error + Send + Sync>>;
```

Comment:
Can we give some comments for these methods?

Reply:
Ok.